Python/prompt-sdk-headless/src/main.py

```diff
@@ -21,6 +21,7 @@ def main():
         params = dict(
             model=prompt.params.model,
             temperature=prompt.params.temperature,
+            max_tokens=prompt.params.max_tokens,
```

Comment:
what is the issue?

Reply:
it needs to be an int, not a float. we could convert here, but I think we should update so you don't have to

Reply:
https://github.com/autoblocksai/autoblocks-examples/actions/runs/7834014036/job/21376333716?pr=97

Reply:
ah ok, yeah we can update here: https://github.com/autoblocksai/python-sdk/blob/fdffc08242bd8f8999bd25550b8c439c077e84b5/autoblocks/_impl/prompts/autogenerate.py#L130-L131

Reply:
cool ya i'll update

Reply:
we need more robustness here in general. probably the types can be stored alongside the model params themselves rather than making each SDK infer them. for example we can't handle empty lists atm